Editor/Window/ConnectToFleetInput.cs

```diff
@@ -103,15 +103,16 @@ private void OnCancelButtonClicked()
             UpdateGUI();
         }
 
-        private void OnSelectFleetDropdown(string fleetName)
+        private async void OnSelectFleetDropdown(string fleetName)
         {
             var currentFleet = _fleetAttributes.FirstOrDefault(fleet => fleet.Name == fleetName);
             if (currentFleet != null)
             {
+                var fleetLocationResponse = await _fleetManager.DescribeFleetLocationAttributes(currentFleet.FleetId);
                 _fleetIdText.text = currentFleet.FleetId;
                 _stateManager.AnywhereFleetName = currentFleet.Name;
                 _stateManager.AnywhereFleetId = currentFleet.FleetId;
-                _stateManager.AnywhereFleetLocation = _fleetManager.FleetLocation;
```

Comment:
Does `FleetLocation` still need to be public? Can we revert that now that we no longer reference it

Reply:
It did have a use above here but I have also added the sdk call there as well, then changed fleetlocation back to private.

Reply:
I don't understand what you mean here, there's no other FleetLocation reference that you removed. I presume you're referring to the `OnAnywhereConnectClicked`, but that creates the location to use so there's no reason to find a location.